packages/plugin-ext/doc/how-to-add-new-custom-plugin-api.md

```diff
@@ -89,7 +89,6 @@ import { FooExtImpl } from './foo-ext-impl';
 import * as fooBarAPI from '@bar/foo';
 
 type FooBarApi = typeof fooBarAPI;
-type Foo = FooBarApi['Foo'];
```

Comment:
This type is used at line 132.

Reply:
Indeed. I apparently got myself confused there. Will fix